test/parallel/test-stream-readable-emittedReadable.js

```diff
@@ -43,12 +43,22 @@ const noRead = new Readable({
   read: () => {}
 });
 
-noRead.on('readable', common.mustCall(() => {
+noRead.once('readable', common.mustCall(() => {
   // emittedReadable should be true when the readable event is emitted
   assert.strictEqual(noRead._readableState.emittedReadable, true);
   noRead.read(0);
   // emittedReadable is not reset during read(0)
   assert.strictEqual(noRead._readableState.emittedReadable, true);
+
+  noRead.once('readable', common.mustCall(() => {
```

Comment:
By using `once` here we do not test that `'readable'` is emitted exactly two times. Is this wanted?

Reply:
Done.